service/src/main/kotlin/fi/espoo/evaka/invoicing/controller/IncomeController.kt

```diff
@@ -51,7 +52,11 @@ class IncomeController(
     @GetMapping
     fun getIncome(db: Database.Connection, user: AuthenticatedUser, @RequestParam personId: PersonId): Wrapper<List<Income>> {
         Audit.PersonIncomeRead.log(targetId = personId)
-        accessControl.requirePermissionFor(user, Action.Person.READ_INCOME, personId)
+        if (!accessControl.hasPermissionFor(user, Action.Person.READ_INCOME, personId) &&
+            !accessControl.hasPermissionFor(user, Action.Child.READ_INCOME, personId)
+        ) {
+            throw Forbidden("Permission denied")
+        }
```

Comment:
I don't think a separate `Child.READ_INCOME` action is necessary so this check could be left as is.

Reply:
Yeah, I trusted too much to type in `ChildInformation.tsx#requireOneOfPermittedActions`. I guess https://github.com/espoon-voltti/evaka/pull/1759/commits/1996cce57a7c842ec6eb6ffb94a55f90dfde9fbc is the correct fix?